server/files/javascript/new.js

```diff
@@ -431,6 +431,21 @@ semantic.new.ready = function() {
       max: 10,
       start: 5
     });
+
+  $('#__twoseven .ui.dropdown').dropdown();
```

Comment:
You could change `#__twoseven` to `.twoseven` and then you wouldn't need to add the ID to the div either

Reply:
Addressed in https://github.com/fomantic/Fomantic-UI-Docs/pull/130/commits/d3c373f22caba7217ffd4ccb15b6d18f33100e9b
There is no `.twoseven` but `div[data-tab="twoseven"]`.